### ISSUE TITLE
App crashes on attempting to load particular new chapter

### ISSUE THREAD

**Issue by fuzxi** [tachiyomiorg/tachiyomi#7891]
### Steps to reproduce

Open specific chapter from chapter list, or open previous chapter and scroll to the page separating it from the specific chapter.

### Expected behavior

Chapter should open.

### Actual behavior

Loading circle appears as chapter loads, then screen turns gray and chapter list is reopened.

### Crash logs

[tachiyomi_crash_logs.txt](https://github.com/tachiyomiorg/tachiyomi/files/9443661/tachiyomi_crash_logs.txt)

### Tachiyomi version

0.13.6

### Android version

Android 12 (CalyxOS)

### Device

Google Pixel 4a

### Other details

This only occurs when opening chapter 76 of [Jahy-sama on MangaDex.](https://mangadex.org/chapter/1f754677-8a5b-439c-8613-fb7252587f56/1) It occurs regardless of whether or not the chapter is downloaded. 
The MangaDex extension version is 1.3.166.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by fuzxi** [comment]
I don't think it is a duplicate. The crash reasons are different, and this is unrelated to webtoons. @Seishirou101 please doublecheck.

**Comment by Seishirou101** [comment]
no its the same crash

**Comment by fuzxi** [comment]
Okay, my bad. Thank you.

**Comment by fuzxi** [comment]
I just realized that I uploaded the wrong crash log. I downloaded the log from #7890 to check if it was the same as mine before submitting the issue and got them mixed up.

This is the correct log: [tachiyomi_crash_logs.txt.](https://github.com/tachiyomiorg/tachiyomi/files/9448464/tachiyomi_crash_logs.txt)

Mind checking one more time please?

**Comment by Seishirou101** [comment]
yes thats a new crash.

**Comment by abhishekabk5** [comment]
I followed the crash how-to (opening chap 76 on jahy-sama on mangadex), got the crash every time (tried 2-3 times). However, after I built and installed the devDebug version (on release v0.13.6 tag) with different applicationId, the chap opened easily on first try and the main app stopped crashing too after that, probably due to some caching.

Just leaving this info here for anyone else looking into this.
